File: Assignment54/Problem3.cpp
#include<iostream>
using namespace std;

void Display( int no)
{
	static int iCnt = no;
	
	if(iCnt > 0)
	{
		cout<<iCnt<<"\t";
		iCnt--;
		Display(no);
	}
}
int main()
{
	int iValue =0;
	
	cout<<"Enter number"<<endl;;
	cin>>iValue;
	
	Display(iValue);
	
	return 0;
}

File: Assignment52/Problem3.java

//	Accept no and and check whether 9th and 12th bit is ON or OFF;

import java.lang.*;
import java.util.*;

class BitWise
{
	public boolean CountOnBit(int iNo)
	{
		int iMask = 0X00000900;
		
			int iResult = iNo & iMask;
			
			if(iResult != 0)
			{
				return true;
			}
			else
			{
				return false;
			}

	}
}

class Problem3
{
	public static void main(String a[])
	{
		Scanner sobj= new Scanner(System.in);
		
		System.out.println("Enter Number");
		int iValue = sobj.nextInt();
		
		BitWise bobj = new BitWise();
		
		boolean iRet = bobj.CountOnBit(iValue);
		
		if(iRet == true)
		{
			System.out.println("Bit is on");
		}
		else
		{
			System.out.println("Bit is off");
		}
	}
	
}